### PR TITLE
Add generic regs.gov link option to RegComment

### PR DESCRIPTION
**Pull request by Scotchester**
Stakeholders would like the link in this sentence to go directly to the comment form on Regs.gov for the specific document:

> If you would like to include an attachment with your response, please submit your comment at [Regulations.gov](https://www.regulations.gov/).

We can't do this ALL the time, because in situations like the Payback Playbook, we may need to publish our commenting form before the document is fully populated with content on Regulatons.gov, and we don't want people going to an empty page. Thus, the need for this new option in the Wagtail module.

## Additions

- RegComment organism: New option to use a generic link for commenting at Regulations.gov or going directly to the specified document's comment form.

## Review

- @kurtw 
- @rosskarchner 
- @kave 
- @richaagarwal 
- @sebworks 
- @anselmbradford 
- @jimmynotjim 
- @KimberlyMunoz 

## Todos

- This option (and the statement as a whole) can be removed when/if we add the ability to submit attachments via our form.

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/v1/models/organisms.py [hunk 1]
```diff
@@ -82,8 +82,11 @@ class Meta:
 class RegComment(blocks.StructBlock):
     document_id = blocks.CharBlock(required=True, label='Document ID',
                                    help_text='Federal Register document ID number to which the comment should be submitted. Should follow this format: CFPB-YYYY-####-####')
-    id = blocks.CharBlock(required=False, label='id',
-                          help_text='Sets the id attribute in the form\'s markup. If not set, the form will be assigned a base id of `o-reg-comment_` with a random number appended.')
+    generic_regs_link = blocks.BooleanBlock(required=False, default=True,
+                                            label='Use generic Regs.gov link?',
+                                            help_text='If unchecked, the link to comment at Regulations.gov if you want to add attachments will link directly to the document given above. Leave this checked if this comment form is being published before the full document is live at Regulations.gov, then uncheck it when the full document has been published.')
```
Comment (sebworks) [inline cfgov/v1/models/organisms.py]: can we make this shorter? This seems a bit long to be helpful

File: cfgov/jinja2/v1/_includes/organisms/reg-comment.html [hunk 2]
```diff
@@ -34,7 +38,12 @@
             If you would like to include an attachment with your response,
             please submit your comment at
             <a class="icon-link icon-link__external-link"
-               href="https://www.regulations.gov/">
+               href="https://www.regulations.gov/{{
+                    '#!submitComment;D=' + value.document_id
```
Comment (sebworks) [inline cfgov/jinja2/v1/_includes/organisms/reg-comment.html]: does the `#!submitComment` need to be within `{{`? Seems like it's static.

Comment (Scotchester) [inline cfgov/jinja2/v1/_includes/organisms/reg-comment.html]: Yes, if `generic_regs_link` is false, the link should just be https://www.regulations.gov/

File: cfgov/v1/models/organisms.py (hunk 1 above)
Comment (Scotchester) [inline cfgov/v1/models/organisms.py]: It is long, but I want to make sure Wagtail users know what the purpose of the option is and be clear on what happens if they unchecked it.

**Comment by kurtw** [comment]
Looks good to me 👍

**Comment by sebworks** [comment]
👍

**Pull request merged**